### ISSUE TITLE
AWS Elastic Beanstalk deployment failing

### ISSUE THREAD

**Issue by jarekkantor** [travis-ci/dpl#761]
We've noticed that our deployments to Elastic Beanstalk started to fail on 1st of March.

Here's deploy section of travis config:
```
deploy:
  - provider: elasticbeanstalk
    edge: true
    skip_cleanup: true
    zip_file: 'build.zip'
    access_key_id: _access_key_id_
    secret_access_key:
      secure: _secret_access_key_
    region: ap-southeast-2
    app: app
    env: production
    bucket_name: deployments-prod
    on:
      repo: _repo/repo_
      branch: master
```

Error in Travis:
```
$ rvm $(travis_internal_ruby) --fuzzy do ruby -S gem install dpl --pre
Fetching: dpl-1.9.1.travis.2639.5.gem (100%)
Successfully installed dpl-1.9.1.travis.2639.5
1 gem installed

Installing deploy dependencies
ERROR: Could not find a valid gem 'dpl-elasticbeanstalk' (= 1.9.1.travis.2639.5) in any repository
ERROR: Possible alternatives: elastic-beanstalk, aws-sdk-elasticbeanstalk, asir_beanstalk, dashing-beanstalk, elastic_beans
/home/travis/.rvm/rubies/ruby-2.2.6/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:59:in `require': cannot load such file -- dpl/provider/elastic_beanstalk (LoadError)
from /home/travis/.rvm/rubies/ruby-2.2.6/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:59:in `require'
from /home/travis/.rvm/gems/ruby-2.2.6/gems/dpl-1.9.1.travis.2639.5/lib/dpl/provider.rb:93:in `rescue in block in new'
from /home/travis/.rvm/gems/ruby-2.2.6/gems/dpl-1.9.1.travis.2639.5/lib/dpl/provider.rb:67:in `block in new'
from /home/travis/.rvm/gems/ruby-2.2.6/gems/dpl-1.9.1.travis.2639.5/lib/dpl/cli.rb:41:in `fold'
from /home/travis/.rvm/gems/ruby-2.2.6/gems/dpl-1.9.1.travis.2639.5/lib/dpl/provider.rb:66:in `new'
from /home/travis/.rvm/gems/ruby-2.2.6/gems/dpl-1.9.1.travis.2639.5/lib/dpl/cli.rb:31:in `run'
from /home/travis/.rvm/gems/ruby-2.2.6/gems/dpl-1.9.1.travis.2639.5/lib/dpl/cli.rb:7:in `run'
from /home/travis/.rvm/gems/ruby-2.2.6/gems/dpl-1.9.1.travis.2639.5/bin/dpl:5:in `<top (required)>'
from /home/travis/.rvm/gems/ruby-2.2.6/bin/dpl:23:in `load'
from /home/travis/.rvm/gems/ruby-2.2.6/bin/dpl:23:in `<main>'
failed to deploy

```
Removing line `edge: true` fixed the issue but I think it was introduced in branch v1.9

**Comment by BanzaiMan** [comment]
When reporting a problem, please include a build log URL that shows the problem you are describing. If it’s for a private repository and (or for any reason) you’d rather not discuss it in public, please close this issue and send email to support@travis-ci.com.

Thank you!

**Comment by BanzaiMan** [comment]
This should now be fixed by `1.9.1.travis.2660.5` or later. Please keep in mind that `edge` is unstable.

Let us know if you continue to see problems with later versions.

Thanks.
